### PR TITLE
Improve maintainability by merging Standard.json & LowPri.json

### LINKED ISSUE #21: No RDP port set
I wasn't able to connect with remote desktop.  On closer inspection it doesn't look like port 3389 is set in the NAT incoming settings.

### LINKED ISSUE #26: evictionPolicy?
Hi
After deleting VM I've lost whole setup. What do you think to change "evictionPolicy" to "deallocate"?

### LINKED ISSUE #36: VM from template - nvidia driver not properly installed
Yesterday I deployed NV6 directly from the template link - single VM, standard (not spot), after coming up there was Steam and zerotier properly installed however there was no disconnect shortcut on desktop. So I made it myself but even after changing 2 to 1 or vice versa so that it disconnected me when clicking it, steamlink connection showed microsoft driver and everything was slow. also the custom extension in portal showed some error messages related to nvidia driver. Looks like it was  expected in some 2nd level path on C: but it was actually deeply nested in like 5 folders. Anyway I rerun it in same way (with silent noreboot flags) then rebooted  and it didn't help. So I run some stuff semi-randomly from setup and setup2 (like e.g. disable hyperv driver) rebooted again and now it finally works great.
 
So just a report that it failed out of box for me yesterday. Maybe current nvidia driver is extracted into wrong location/has different folder structure than expected?

Oh, one more thing, there was no NSG set for the VM networking so the RDP port  was open to whole world without any hint anywhere in portal. So might be better to include NSG in  the template just so that people see more clearly the rdp port is open to whole world and may secure it properly or even disable it completely when using zerotier.

BTW I was using this like year ago with low priority VM scaleset and it worked great but now with spot instances this is so unstable I went for normal VM now. Actually I didn't use it few months and the scaleset was somehow automatically converted by MS from low priority VM to spot instance version with policy set as delete(!). So yesterday I started it for the first time after months and  was playing some game and it suddenly disconnected an I only managed to see 'deleting' in portal and whole vm was gone! Only then I noticed the policy of course. I also had other non-steam games there so lost many save games progress this way :-(

Anyway thanks for this, I am using steamlink from raspberry pi with this and enjoy absolutely quiet high quality gaming :-) Well sometimes I die because of sudden lag spike but it is not that bad.

### PR DESCRIPTION
**Pull request by AmineI**
Adds Spot VM as a param in Standard.json. This gets rid of the need to use a different template for LowPri, 

I was wondering why LowPri was using VM Scale Sets, and I think it was an older limitation of LowPri before it became Spot. If there are advantages I don't know of, I'd be glad to be enlightened :).

However, now Azure Spot VM can be deployed without Scale Sets, so merging both templates would really help with maintainability down the road, and eliminate the need to test the Spot deployments in the future as they would have no difference in configuration.

______

- [x] Standard with the Spot VM parameter disabled works properly.
- [x] Standard with the Spot VM parameter enabled works properly. Thanks to @andiradulescu for his help testing this.
- [x] Readme updates

[Deployment link for this branch](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAmineI%2Fazure-gaming%2Fmerge-LowPri%2FStandard.json) 

______
Fixes #36 (the second issue mentioned in it, due to changing the evictionpolicy to deallocate)
Fixes #26
Fixes #21 (LowPri now have the same RDP config of Standard, as a result of the merge)

### CODE REVIEW

**Comment by andiradulescu** [comment]
@AmineI - testing this now, with Standard_NV6.

Running the template with `Standard_NV6_Promo` gives this error:

```
{
    "error": {
        "code": "OperationNotAllowed",
        "message": "The requested VM Size Standard_NV6_Promo does not support creation of Low Priority instances."
    }
}
```

**Comment by AmineI** [comment]
Yes, Promo is not supported in LowPri indeed. You fixed that in #39 but that was when there were 2 deployment files.. I'll see if I can hide Promo when Spot is selected

**Comment by andiradulescu** [comment]
With Standard_NV6  everything seems to work fine! Steam and the NVIDIA drivers are installed.

**Comment by AmineI** [comment]
> With Standard_NV6 everything seems to work fine! Steam and the NVIDIA drivers are installed.

Great, thanks ! Steam and Nvidia drivers matter less here as I'm merging against master, which does not have the changes from #41 - only the VM deployment matters for this PR. 

>Yes, indeed, not sure if you can hide the Promo one when there is only one template.
Yup. The error message seems explicit enough so it should be fine this way, with a small note in the readme.

I made a last set of minor update to the "Spot Disabled" behavior, and reconfirmed the non-spot deployment. The template should be ready to go, and I'll tackle the readme update once ecalder6 validates the template changes.

File: Standard.json [hunk 1]
```diff
@@ -25,6 +25,13 @@
                 "description": "The type of the VM to deploy. Note that only NV-series support gaming."
             }
         },
+        "isSpotVM": {
+            "type": "bool",
+            "metadata": {
+                "description": "Whether the VM is a Spot VM or not. Spot VMs let you use unused Azure resources for a cheaper price but can get force stopped at anytime."
```
Comment (ecalder6) [inline Standard.json:31]: Recommend adding a note saying that spot does not work with promo skus.

File: Standard.json [hunk 2]
```diff
@@ -212,7 +219,10 @@
                             "id": "[resourceId('Microsoft.Network/networkInterfaces',variables('nicName'))]"
                         }
                     ]
-                }
+                },
+                "priority": "[if(parameters('isSpotVM'), 'Spot', 'Regular')]",
```
Comment (ecalder6) [inline Standard.json:223]: This is great!

File: Standard.json (hunk 1 above)
Comment (AmineI) [reply Standard.json]: Done !

File: README.md [hunk 3]
```diff
@@ -140,10 +123,11 @@ In Steam Remote Play, you can toggle streaming stats display with F6.
 
 #### I Want to Manually Deploy My VM
 
-You could manually deploy your VM through Azure portal, PowerShell, or Azure CLI. Note that a low priority VM has to exist in a virtual machine scale set, making it a bit more difficult to do manually.
+You could manually deploy your VM through Azure portal, PowerShell, or Azure CLI. 
+
+1. Deploy a NV6 size VM through the azure portal(see [this guide](https://lg.io/2016/10/12/cloudy-gamer-playing-overwatch-on-azures-new-monster-gpu-instances.html) for instructions).
+2. Remote desktop into your Azure VM instance.
 
-1. Deploy a NV6 size VM through the azure portal(see [this guide](https://lg.io/2016/10/12/cloudy-gamer-playing-overwatch-on-azures-new-monster-gpu-instances.html) for instructions). For low priority VM, checkout [this documentation](https://docs.microsoft.com/en-us/azure/virtual-machine-scale-sets/virtual-machine-scale-sets-use-low-priority) and [this reddit thread](https://www.reddit.com/r/cloudygamer/comments/8fu2d0/azure_pricing_update/). Do not forget to add the Nvidia Driver Extension to the VM !
```
Comment (ecalder6) [inline README.md:145]: I think you accidentally removed "Do not forget to add the Nvidia Driver Extension to the VM !"

File: README.md [hunk 4]
```diff
@@ -140,10 +123,11 @@ In Steam Remote Play, you can toggle streaming stats display with F6.
 
 #### I Want to Manually Deploy My VM
 
-You could manually deploy your VM through Azure portal, PowerShell, or Azure CLI. Note that a low priority VM has to exist in a virtual machine scale set, making it a bit more difficult to do manually.
+You could manually deploy your VM through Azure portal, PowerShell, or Azure CLI. 
+
+1. Deploy a NV6 size VM through the azure portal(see [this guide](https://lg.io/2016/10/12/cloudy-gamer-playing-overwatch-on-azures-new-monster-gpu-instances.html) for instructions).
```
Comment (AmineI) [inline README.md:128]:
```suggestion
1. Deploy a NV6 size VM through the azure portal(see [this guide](https://lg.io/2016/10/12/cloudy-gamer-playing-overwatch-on-azures-new-monster-gpu-instances.html) for instructions). Do not forget to add the Nvidia Driver Extension to the VM !
```

File: README.md (hunk 3 above)
Comment (AmineI) [reply README.md:145]: Correct, missed it in the merge conflict ! Fixed it :)

**Pull request merged**